src/3rdparty/webkit/Source/WebCore/platform/qt/PlatformScreenQt.cpp

```diff
@@ -39,9 +39,40 @@
 #include "QWebPageClient.h"
 #include <QApplication>
 #include <QDesktopWidget>
+#include <QByteArray>
 
 namespace WebCore {
 
+static unsigned long qt_get_screen_width() {
+  QByteArray width = qgetenv("WKHTMLTOX_SCREEN_WIDTH");
+  unsigned long screen_width = 1366;
+
+  bool ok;
+  unsigned long screen_width_tmp = width.toULong(&ok, 10);
+  if (ok == true) {
+      if ((screen_width_tmp >= 320) && (screen_width_tmp <= 7680)) {
+         return screen_width_tmp;
+      }
+  }
```

Comment:
Formatting is not correct (2 spaces vs 4 spaces). Also, would prefer simpler code similar to the following (untested):
```c
static unsigned int wkhtmltox_screen_width()
{
    QByteArray override = qgetenv("WKHTMLTOX_SCREEN_WIDTH");

    bool ok;
    unsigned int width = override.toUInt(&ok);
    if (ok)
        return qMax(320, qMin(7680, width));

    return 1366; // default screen width
}
```
Screen resolution is not going to exceed `int` and ideally the function name should be similar to the environment variable.